package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "moodle-repository_ocis",
```

Comment:
how about moving all root level test related files to `tests/cucumber/acceptance`? Just my opinion